rust/src/lib.rs

```diff
@@ -49,6 +49,10 @@
 // just due to FFI.
 #![allow(clippy::missing_safety_doc)]
 
+// Allow /// cbindgen:ignore comments on extern blocks
+// cf https://github.com/mozilla/cbindgen/issues/709
+#![allow(unused_doc_comments)]
```

Comment:
Annoying that this is required.

Reply:
Maybe we can have these individually and not globally..?